.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+<!--
+Thanks for submitting a Pull Request!
+
+Please shortly explain your contribution, and if fixing an issue from the tracker, please add "Fixes #XXX", replacing XXX with the issue number.
+
+Be sure that your contribution follows the [Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/).
+
+Be sure to go over each item in the list below before submitting your pull request.
+-->
+
+### Description
+
+Briefly describe what your changes are doing. It is more important to mention why the changes are necessary, rather than the how.
+
+### Checklist
+
+- [ ] The commit messages include a [DCO](https://discourse.ros.org/t/starting-to-enforce-developer-certificate-of-origin-dco-for-some-ros-2-repos/7420).
+- [ ] The commit messages follow [good practices](https://chris.beams.io/posts/git-commit/).
+- [ ] Tests have been added or updated to cover the changes. If tests cannot be added for some reason, it has been explained in the description.
+- [ ] The changes passes tests locally (see [here](https://index.ros.org/doc/ros2/Tutorials/Colcon-Tutorial) for how to run tests.
+- [ ] The PR is minimal and any unrelated changes are submitted separately.
+- [ ] The API documentation has been updated based on changes in APIs or addition of new APIs, if any.
+- [ ] The [documentation](https://index.ros.org/doc/ros2/) has been updated to describe the new feature or package added, if any.
+
+- Reviewing pull requests takes time and effort. While we are reviewing your pull request, please consider supporting the community by reviewing another open pull request. Refer to the [ROS code review guide](https://github.com/rosin-project/ros_code_review_guide/blob/master/README.md) for general reviewing guidelines and [Quality Guide: Ensuring code quality](https://index.ros.org/doc/ros2/Contributing/Quality-Guide/) for ROS 2 specific guidelines.
+  - [ ] I will review (or have reviewed) another person’s PR. PR tracker No.: ___________________
+  - [ ] I’m willing to review another person’s PR and would like a suggestion.
+  - [ ] I do not want to review another person’s PR.
```

Comment:
Is this something that could be put in a comment rather than in the PR body?  I appreciate the idea of getting other people to contribute to reviewing PRs, but I would be concerned that this would be a lot of noise when going through many PRs.

Reply:
> Is this something that could be put in a comment rather than in the PR body? I appreciate the idea of getting other people to contribute to reviewing PRs, but I would be concerned that this would be a lot of noise when going through many PRs.

We discussed this in the ROS Quality Working group and we thought that static comments tend to be ignored by developers. Hence, we suggested something actionable to incite the contributor to participate in reviewing other PRs.

Reply:
It should be mentioned that a similar pattern is already used by other repositories, most notably [ros-planning/moveit](https://github.com/ros-planning/moveit/blob/master/.github/PULL_REQUEST_TEMPLATE.md).